src/logaze/openapi.clj

```diff
@@ -55,7 +56,7 @@
                        :query-params {:params query}
                        :headers {:referer "https://www.lenovo.com/"}}
                       page-client)]
-     (println {:getting :page :n n})
+     (h/safe-println "getting page" {:n n})
```

Comment:
I actually intended to use structured logging, for the time in the future where I might wanna parse the logs :P

`safe-println` is good, but can we restore the printing to printing one map please?

Reply:
Oh sure. Done.